python/pyarrow/tests/test_flight.py

```diff
@@ -1589,6 +1602,10 @@ def test_roundtrip_types():
     assert info.total_records == info2.total_records
     assert info.endpoints == info2.endpoints
 
+    endpoint = flight.FlightEndpoint(
+        ticket, ['grpc://test', flight.Location.for_grpc_tcp('localhost', 5005)])
```

Comment:
The linter shows this line is too long: https://github.com/apache/arrow/runs/8306752201?check_suite_focus=true#step:5:974

Reply:
yep, did saw this before, my bad. 
should have checked with archery after rebase.